src/main/java/org/isf/medicalinventory/gui/InventoryWardBrowser.java

```diff
@@ -106,16 +128,54 @@ private void initComponents() {
         panelFooter = getPanelFooter();
         getContentPane().add(panelFooter, BorderLayout.SOUTH);
 
-        for (int i = 0; i < pColumwidth.length; i++) {
-            jTableInventory.getColumnModel().getColumn(i).setMinWidth(pColumwidth[i]);
-        }
+        ajustWidth();
 
         addWindowListener(new WindowAdapter() {
             public void windowClosing(WindowEvent e) {
                 dispose();
             }
         });
-        // pack();
```

Comment:
The formatting of the following lines is off; see the screen shot taken from how github sees the file.

![image](https://github.com/user-attachments/assets/fffd0af6-4244-4c4a-83a3-c1c8a0d1b8ae)

Reply:
I am confused because when I checked on my GitHub the formatting was good
![image](https://github.com/user-attachments/assets/d738b8ab-1aa2-4113-961e-e471c1487b65)

Reply:
I think this issue is related to the GitHub default tab (default value = 8) because I changed the tab default value to 4 in the GitHub setting, and now the indention looks good.
Before
![image](https://github.com/user-attachments/assets/24864143-8550-4b96-b19d-c94da823986e)
After ========================================================================
![image](https://github.com/user-attachments/assets/4c485acf-8713-4c0b-850f-a40256cfca52)

Reply:
Sorry, where do you find the GitHub setting for indentation?

Reply:
In the Appearance menu, at the end of the page

Reply:
I see, but the problem is generated by the fact that your code has a mix of tabbed and spaced indentations, so if you ask GitHub to render tabs in a different way of course it appears correct in some cases (i.e. when you have same number of spaces as indentation instead of tabs) but it is just a rendering option.

In Eclipse, this is piloted by this setting "Use spaces to indent wrapped lines" that might be called differently in other IDEs:

without "Use spaces to indent wrapped lines"
![image](https://github.com/user-attachments/assets/5e56b048-a8f8-422d-99a2-e8032035b98d)

with "Use spaces to indent wrapped lines" (mixed indentation)
![image](https://github.com/user-attachments/assets/9bf0532f-7c57-4aec-a99a-84a9d896209c)

I think we should use "Tabs only" setting and to avoid "Use spaces to indent wrapped lines".
Otherwise we may opt for "Spaces only" and forget abobut tabs, but at this point it would mean to reformat the whole code...

![image](https://github.com/user-attachments/assets/36b2d16d-0db6-49bb-b526-a556f68ed85c)


@dbmalkovsky wdyt?